### DISCUSSION TITLE
Validation based on field value (Joi)

### DISCUSSION THREAD

**Discussion by bianchi** [react-hook-form/react-hook-form#3747] [Q&A]
I have a validation rule, using Joi, which should be based on another input:

 `minimum_inventory: Joi.number().precision(**value selected in another input**).required().messages({
				'any.required': 'O estoque mínimo é obrigatório',
				'number.base': 'O estoque mínimo deve ser um número',
			}),`

First thing that came to my mind was to use watch, but validation schema is defined before I can declare a watch variable. Is there a way to change schema validation before validate? Any suggestions?

**Comment by bluebill1049** [comment]
React Hook Form focus on a single input validation at a time, for depend validation you can use `triger`
